README.md

```diff
@@ -103,6 +103,26 @@ returns a body that looks like:
     "three": "value3"
 }
 ```
+Be careful, if you use name of the parameter equal field name from xml,json,key/value body, the URL Pattern Matching parameter will replace this field.
```

Comment:
I had a little trouble understanding this particular warning...
Maybe something like 
`Be careful, when using 'urlRegex', the value captured by the named group in the regex will take precedence over the variables in the xml/json/key-value request body` would be more clear?